src/python/pants/backend/jvm/tasks/ivy_outdated.py

```diff
@@ -67,7 +67,7 @@ def _is_update_coordinate(self, coordinate):
     def execute(self):
         deprecated_conditional(
             lambda: True,
-            removal_version="1.30.0.dev0",
+            removal_version="1.30.1.dev0",
```

Comment:
Do we plan to have a 1.30.1 release? Do you mean 1.31.0?

Reply:
Ooops, yes.